web/static/js/components/remote_retro.jsx

```diff
@@ -6,6 +6,7 @@ import * as AppPropTypes from "../prop_types"
 import Room from "./room"
 import Alert from "./alert"
 import DoorChime from "./door_chime"
+import CATEGORIES from "../configs/categories"
```

Comment:
@tnewell5, rather than house this in its own file, let's juse export `categories` as a constant from the `retro_configs.js` file

Reply:
will do!